ext/openssl/openssl.stub.php

```diff
@@ -433,9 +433,9 @@ function openssl_csr_export(OpenSSLCertificateSigningRequest|string $csr, &$outp
 function openssl_csr_sign(OpenSSLCertificateSigningRequest|string $csr, OpenSSLCertificate|string|null $ca_certificate, #[\SensitiveParameter] $private_key, int $days, ?array $options = null, int $serial = 0): OpenSSLCertificate|false {}
 
 /**
- * @param OpenSSLAsymmetricKey $private_key
+ * @param OpenSSLAsymmetricKey|null $private_key
  */
-function openssl_csr_new(array $distinguished_names, #[\SensitiveParameter] &$private_key, ?array $options = null, ?array $extra_attributes = null): OpenSSLCertificateSigningRequest|false {}
+function openssl_csr_new(array $distinguished_names, #[\SensitiveParameter] $private_key, ?array $options = null, ?array $extra_attributes = null): OpenSSLCertificateSigningRequest|bool {}
```

Comment:
Return type looks right, but the `$private_key` parameter appears to be used as a reference: https://github.com/php/php-src/blob/e0a7ec22ee8962a3273cb8a5da86030932b1876d/ext/openssl/openssl.c#L3461

Reply:
Thanks, you are, of course, right. Completely missed that part...